Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -938,5 +938,13 @@ public string GetFullPath(string filePath)
 
             return _project.Location + Path.DirectorySeparatorChar + filePath;
         }
+
+        public string ProjectName
```

Comment:
I don't think we should add a ProjectName param for no reason, there are already 2 other ways to get it:
1. Check root node
2. Path.GetFilenameWithoutExtension(_controller.ProjectLocation)